tests/benchdnn/graph/ref_partition.cpp

```diff
@@ -87,6 +87,9 @@ int ref_partition_t::init_ref(
         SAFE(ref_prim->init_prb(res), WARN);
 
         SAFE_V(ref_prim->init_prim(::get_test_engine(), res));
+        // reset res to avoid a skipped state from init_prim() to affect the rest part.
+        res->state = UNTESTED;
+        res->reason.clear();
```

Comment:
Discussed offline: unimplemented should be emerged here and processed later the stack where it's known if there's a native driver support or not. If not, unimplemented should be returned to the user.

Adding a skip condition here would be good, too.

Reply:
Thanks for the comments, fixed. The skip condition is already inside the init_prim function ([here](https://github.com/uxlfoundation/oneDNN/blob/f23b3fbd2411f14a64b3f3090b52e2a63a2a4cbb/tests/benchdnn/graph/ref_primitive.cpp#L217)), so I think there's no need to add a new one.

Reply:
It would prevent from going further is SKIPPED status was issued.